actors/builtin/miner/miner_actor.go

```diff
@@ -36,7 +36,7 @@ const (
 type CronEventPayload struct {
 	EventType CronEventType
 	// TODO: replace sectors with a bitfield
-	Sectors []abi.SectorNumber // Empty for global events, such as SurprisePoSt expiration.
+	Sectors *abi.BitField
```

Comment:
Remove the TODO.

Everywhere else, we avoid pointers in chain state. Is this different somehow? Am I right assuming it serializes the same either way?

Reply:
It is.